atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -71,15 +72,41 @@ private CassandraKeyValueServices() {
     }
 
     /**
-     * Attempt to wait until nodes' schema versions match.
+     * Wraps the execution of the specified task with checks verifying that schema versions on Cassandra nodes are
```

Comment:
nit: I get that `wraps` implies enclosing, but would be prefer to be more explicit and say something like "performs checks both before and after ..." as I think it's reasonable in a computing context to have wrappers only intercept things on one side.